pyproject.toml

```diff
@@ -239,3 +246,7 @@ dev = [
     "pyright",
     "pytest-cov>=6.0.0",
 ]
+test = [
```

Comment:
what is the advantage of introducing a new dependency-group? I don't see that group being used/referenced anywhere.